tutorials/editor/using_the_android_editor.rst

```diff
@@ -21,14 +21,37 @@ The Android editor requires devices running Android 5 Lollipop or higher, with a
+- `REQUEST_INSTALL_PACKAGES <https://developer.android.com/reference/android/Manifest.permission#REQUEST_INSTALL_PACKAGES>`__: Enables the editor to install exported project APKs.
+- `RECORD_AUDIO <https://developer.android.com/reference/android/Manifest.permission#RECORD_AUDIO>`__: Requested when the `audio/driver/enable_input <https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-property-audio-driver-enable-input>`__ project setting is enabled.
+
+Tips & Tricks
+-------------
+
+**Input**
+
+- For the best experience and high level of productivity, connecting a bluetooth keyboard & mouse is recommended to interact with the Android editor. The Android editor supports all of the `usual shortcuts and key mappings <https://docs.godotengine.org/en/stable/tutorials/editor/default_key_mapping.html>`__.
+- When interacting with keyboard & mouse, you can decrease the size of the scrollbar using the `interface/touchscreen/increase_scrollbar_touch_area <https://docs.godotengine.org/en/stable/classes/class_editorsettings.html#class-editorsettings-property-interface-touchscreen-increase-scrollbar-touch-area>`__ editor setting.
+- For 2D projects, the `block coding plugin <https://godotengine.org/asset-library/asset/3095>`__ can provide a block-based visual alternative to composing scripts when lacking a connected hardware keyboard.
+
+**Multi-tasking**
+
+- On smaller devices, enabling and using picture-in-picture (PiP) mode provides the ability to easily transition between the *Editor* and the *Play window*.
+
+  - PiP can be enabled via the `run/window_placement/play_window_pip_mode <https://docs.godotengine.org/en/latest/classes/class_editorsettings.html#class-editorsettings-property-run-window-placement-play-window-pip-mode>`__ editor setting.
+  - The `run/window_placement/android_window <https://docs.godotengine.org/en/latest/classes/class_editorsettings.html#class-editorsettings-property-run-window-placement-android-window>`__ editor setting can be used to specify whether the *Play* window should always launch in PiP mode.
+  - **Note:** In PiP mode, the *Play* window does not have input access.
+
+**Projects sync**
+
+- Syncing projects via Git can be done by downloading an Android Git client. We recommend the `Termux terminal <https://termux.dev/en/>`__, an Android terminal emulator which provides access to common terminal utilities such Git and SSH.
+
+  - **Note:** To use Git with the Termux terminal, you'll need to grant *WRITE* permission to the terminal. This can be done by `running the following command <https://wiki.termux.com/wiki/Termux-setup-storage>`__ from within the terminal: ``termux-setup-storage``
```

Comment:
```suggestion
  - **Note:** To use Git with the Termux terminal, you'll need to grant *WRITE* permission to the terminal.
    This can be done by `running the following command <https://wiki.termux.com/wiki/Termux-setup-storage>`__ from within the terminal: ``termux-setup-storage``
```